app/src/app/app.component.html

```diff
@@ -39,31 +56,56 @@
         </button>
       </mat-menu>
     </div>
-  </mat-drawer>
+    <div class="side-options-container">
+      <div
+        (click)="setRoute('defaults')"
+        class="side-option"
+        [ngClass]="{ selected: currentRoute === 'defaults' }"
+        id="about-default-files-button"
+      >
+        <mat-icon>api</mat-icon>
+        <p>
+          About Default files
+        </p>
+      </div>
+      <div
+        (click)="setRoute('about')"
+        class="side-option"
+        [ngClass]="{ selected: currentRoute === 'about' }"
+        id="about-spec-math-button"
+      >
+        <mat-icon>info</mat-icon>
+        <p>
+          About Spec Math
+        </p>
+      </div>
+    </div>
+  </mat-sidenav>
 
-  <mat-drawer-content id="main-container" class="main right-panel padded">
-    <div *ngIf="!operationSet.valid" id="empty-text-container">
+  <mat-sidenav-content id="main-container" class="main right-panel padded">
+    <div *ngIf="currentRoute === 'home'" id="empty-page-container">
       <h2 class="mat-h2 mat-title">
         You haven't created any new specifications yet
       </h2>
       <h4 class="mat-h4 mat-subheading-1">
-        Use the "New" button to get started
+        Use the {{ snav.opened ? '"New" button' : "menu" }} to get started
       </h4>
     </div>
```

Comment:
Suggest making a simple component for this so it can be associated to the home path and included in the router outlet.

Would also allow you to conditionally show this in the appDisplayResutls component based on operationSet.valid (it looks like a redirect is the current solution for showing this when !operationSet.valid and the path is /results).

Reply:
Put the "empty" page into its own component since its now being used directly by the routing service instead of *ngIfs.